src/program/services/scrapers/shared.py

```
@@ -46,34 +47,76 @@ def _parse_results(
+                if "resolutions" in ranking_overrides and ranking_overrides["resolutions"] is not None:
+                    resolutions_list = ranking_overrides["resolutions"]
+                    if resolutions_list:
+                        # Reset all to False
+                        for res_key in overridden_settings.resolutions.model_fields:
+                            setattr(overridden_settings.resolutions, res_key, False)
+                        # Enable selected
+                        for res_key in resolutions_list:
+                            if hasattr(overridden_settings.resolutions, res_key):
+                                setattr(overridden_settings.resolutions, res_key, True)
+                    
+                    enabled_res = [k for k, v in overridden_settings.resolutions.model_dump().items() if v is True]
```

Ruff (0.14.6)

Local variable `enabled_res` is assigned to but never used

Remove assignment to unused variable `enabled_res`

(F841)